presto-product-tests/bin/run_on_docker.sh

```diff
@@ -142,12 +142,16 @@ fi
 # catch terminate signals
 trap terminate INT TERM EXIT
 
-if [[ "$ENVIRONMENT" == "singlenode" || "$ENVIRONMENT" == "multinode" ]]; then
-  EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra"
-elif [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
+if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
   EXTERNAL_SERVICES="hadoop-master sqlserver"
 elif [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
   EXTERNAL_SERVICES="hadoop-master ldapserver"
+elif [[ "$ENVIRONMENT" == "singlenode-mysql" ]]; then
+  EXTERNAL_SERVICES="hadoop-master mysql"
+elif [[ "$ENVIRONMENT" == "singlenode-postgresql" ]]; then
+  EXTERNAL_SERVICES="hadoop-master postgresql"
```

Comment:
do we really need hadoop when testing psql?

Reply:
ditto